AEPAssurance/Source/PincodeEntry/SessionAuthorizingUI.swift

```diff
@@ -27,13 +27,13 @@ protocol SessionAuthorizingUI {
     mutating func show(callback : @escaping PinCodeCallback)
 
     /// Invoked when the a socket connection is initialized
-    func connectionInitialized()
+    func onSessionInitialized()
```

Comment:
on* naming is not "swifty" -- i'd recommend starting all protocol methods with "session" and pass the session to the method:

e.g.

sessionInitialized(session: AssuranceSession)

... repeat for all methods in this protocol.

Reply:
👍🏼  removed `on` in all of these methods

Also on these protocol methods we don't have requirement to pass the  `AssuranceSession`. So am still keeping the parameter's empty